spec/tapioca/addon_spec.rb

```diff
@@ -0,0 +1,53 @@
+# typed: true
+# frozen_string_literal: true
+
+require "addon_spec_helper"
+require "ruby_lsp/ruby_lsp_rails/runner_client"
+require "minitest/hooks"
+
+module RubyLsp
+  module Tapioca
+    class AddonSpec < Minitest::HooksSpec
+      # The approach here is based on tests within the Ruby LSP Rails gem
+
+      before(:all) do
+        FileUtils.cp("spec/dummy/bin/rails", "bin/rails")
+        @outgoing_queue = Thread::Queue.new
+        @client = T.let(RubyLsp::Rails::RunnerClient.new(@outgoing_queue), RubyLsp::Rails::RunnerClient)
+      end
+
+      after(:all) do
+        @client.shutdown
+
+        # On Windows, the server process sometimes takes a lot longer to shutdown and may end up getting force killed,
+        # which makes this assertion flaky
+        assert_predicate(@client, :stopped?) unless Gem.win_platform?
```

Comment:
We don't develop or run CI on windows so I think we can remove this. Unless you think it's worth it for external contributors.

Reply:
I'll remove. I think it's unlikely that someone using Windows would want to contribute to developer tooling that doesn't run on Windows.